packages/oruga-next/src/components/input/Input.vue

```diff
@@ -396,6 +400,7 @@ const rootClasses = defineClasses(
         null,
         computed(() => props.expanded),
     ],
+    ["hasIconRightClass", "", null, computed(() => hasIconRight.value)],
```

Comment:
I would suggest something more general like `hasIconClass` with a given suffix like `[left]`, `[right]` or `[left,right]` or something like that. The suffix can also be just a string with one word `left`, `right`, `both` if an array is not suitable.